media-tv/kodi/kodi-17.3.ebuild

```diff
@@ -0,0 +1,279 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+# Does not work with py3 here
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite"
+
+inherit eutils linux-info python-single-r1 cmake-utils autotools
```

Comment:
Lets get the inherit for pax so we can pax-mark kodi.bin with emulate trampolines for hardened support